src/app/coaching-sessions/[id]/page.tsx

```diff
@@ -183,13 +74,7 @@ export default function CoachingSessionsPage() {
               </TabsList>
               <TabsContent value="notes">
                 <div className="flex-col h-full space-y-4">
-                  <CoachingNotes
-                    ref={editorRef}
-                    value={note.body}
-                    onChange={handleOnChange}
-                    onKeyDown={handleKeyDown}
-                  ></CoachingNotes>
-                  <p className="text-sm text-muted-foreground">{syncStatus}</p>
+                  <TipTapEditor />
```

Comment:
I originally had this component called CoachingNotes as a specific design decision, to keep the component used on the page as a generically named one (abstract) that would then instantiate the specific editor under it (concrete) called TipTapEditor.

My reasoning for this is so we can keep the level of tight coupling to TipTap as low as possible. Would you be open to restoring this pattern?

Reply:
Yup! very onboard with this 
2ce66d7266cbbd82f8c34b65e999e51be7acc5a5